README.md

```diff
@@ -112,7 +112,7 @@ impl Default for StatusMessage {
 
 To declare a payable method simply use `#[payable]` decorator:
 ```rust
-user near_sdk::payable;
+use near_sdk::payable;
```

Comment:
I don't think we need this line at all. `#[payable]` is inner proc-macro, so it's not required to be imported. Similar to `#[init]`